SYCL/Basic/fpga_tests/fpga_queue.cpp

```diff
@@ -14,6 +11,7 @@
 //
 //===----------------------------------------------------------------------===//
 #include <CL/sycl.hpp>
+#include <CL/sycl/backend/opencl.hpp>
```

Comment:
According to the SYCL 2020 spec, the OpenCL interop header is named `sycl/backend/opencl.hpp` (without the `CL`).  Do we support that naming?

Reply:
There’s a separate pull request for that change.

Reply:
OK